bootstrap.ps1

```diff
@@ -7,13 +7,28 @@ Push-Location (Join-Path $PSScriptRoot "build")
     .\prerequisites.ps1
 Pop-Location
 
+Push-Location (Join-Path $PSScriptRoot "./src/Simulation/qdk_sim_rs")
+    # We use dotnet-script to inject the version number into Cargo.toml,
+    # so we go on ahead here and restore any missing tools.
+    # Since that Cargo.toml is referenced by CMake lists in the QIR
+    # runtime, this injection has to be the first thing we do.
+    dotnet tool restore
+    dotnet script inject-version.csx -- `
+        --template Cargo.toml.template `
+        --out-path Cargo.toml `
+        --version $Env:NUGET_VERSION;
+Pop-Location
+
 if (-not (Test-Path Env:AGENT_OS)) {
```

Comment:
I just recently added a pre-emptive build of the QIR Runtime to this block (see #706). Just calling that out to make sure that when that change gets merged to the working branch or feature branch it is done in the right order relative to the qdk_sim_rs added below.

Reply:
Looks like catching up to main went cleanly, so will spin up a new build with that change. Thanks!